### PR TITLE
HAAR-884 Refactor files to be user explicit

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/ExternalUserSearchController.kt [hunk 1]
```diff
@@ -15,13 +15,13 @@ import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.manageusersapi.config.ErrorResponse
-import uk.gov.justice.digital.hmpps.manageusersapi.service.UserSearchService
```
Comment (ravmoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/ExternalUserSearchController.kt:18]: Suggestion: is it a good idea to add into packages like : 'nomis' , 'external' under service and resources

**Review by ravmoj — DISMISSED** [review]
lgtm

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/ExternalUserSearchController.kt (hunk 1 above)
Comment (karenmillermoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/ExternalUserSearchController.kt:18]: Yeah - that was the alternative - Happy either way tbh....I'll refactor like that - cheers.

**Review by ravmoj — APPROVED** [review]
lgtm

**Pull request merged**